tests/testthat/helpers.R

```diff
@@ -0,0 +1,10 @@
+skip_if_over_rate_limit <- function(by = 5) {
```

Comment:
Maybe add a note that this is inspired by the function of the same name in `remotes`?